cmd/pitr/recoverer/recoverer.go

```diff
@@ -272,6 +287,31 @@ func getLastBackupGTID(infoObj io.Reader) (string, error) {
 	return string(set), nil
 }
 
+func getDecompressedContent(infoObj io.Reader) ([]byte, error) {
+	tmpDir := os.TempDir()
+
+	cmd := exec.Command("xbstream", "-x", "--decompress")
+	cmd.Dir = tmpDir
```

Comment:
tmpDir used only in one place
let's use it as `cmd.Dir = os.TempDir()`

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/blob/K8SPXC-598/cmd/pitr/recoverer/recoverer.go#L292
https://github.com/percona/percona-xtradb-cluster-operator/blob/K8SPXC-598/cmd/pitr/recoverer/recoverer.go#L305